test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/HangDumpProcessTreeTests.cs

```diff
@@ -52,7 +58,6 @@ public sealed class TestAssetFixture() : TestAssetFixtureBase(AcceptanceFixture.
   <PropertyGroup>
     <TargetFrameworks>$TargetFrameworks$</TargetFrameworks>
     <OutputType>Exe</OutputType>
-    <UseAppHost>true</UseAppHost>
```

Comment:
why is this removed? the global property should override it. Is that to keep it "os specific" if the build does / will do that?